tools/code_format/check_format.py

```diff
@@ -356,17 +358,6 @@ def check_namespace(self, file_path):
             ]
         return []
 
-    def package_name_for_proto(self, file_path):
-        package_name = None
-        error_message = []
-        result = PROTO_PACKAGE_REGEX.search(self.read_file(file_path))
-        if result is not None and len(result.groups()) == 1:
-            package_name = result.group(1)
-        if package_name is None:
-            error_message = ["Unable to find package name for proto file: %s" % file_path]
```

Comment:
i belive that proto_format also effectively checks the package name, but i will check this